.github/workflows/first-interaction.yml

```diff
@@ -0,0 +1,16 @@
+name: Greetings
+
+on: [pull_request, issues]
+
+jobs:
+  greeting:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - uses: actions/first-interaction@v1
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          issue-message: '<h1>It''s great having you contribute to this project</h1> Welcome to the community :nerd_face: <br> If you would like to continue contributing to open source and would like to do it with an awesome community, you should join our <a href="https://discord.gg/3EN2UVE9mH">Discord Server</a> and our <a href="https://github.com/web3community">GitHub Organisation</a> - we help and encourage each other to grow in web3 🤓. Any questions let us know.'
+          pr-message: '<h1>It''s great having you contribute to this project</h1> Welcome to the community :nerd_face: <br> If you would like to continue contributing to open source and would like to do it with an awesome community, you should join our <a href="https://discord.gg/3EN2UVE9mH">Discord Server</a> and our <a href="https://github.com/web3community">GitHub Organisation</a> - we help and encourage each other to grow in web3 🤓. Any questions let us know.'
```

Comment:
I think we should customize it to be another message and not a copy of EddieHub's. 🤔